kiwi_post_run

```diff
@@ -68,6 +68,13 @@ if [ -z "$IMAGE" ]; then
     exit 0
```

Comment:
@jordimassaguerpla I followed the same criteria as here, provide a reasonable message to explain what happened and just exit without doing anything. This code as not done by me, thus I don't know the motivations behind it, but to my understanding it has some sense, because this is a post run hook, meaning that the problem is not with the image, as at these stage it has already been build successfully, thus I think it makes sense to not fail as the image is valid (in KIWI/OBS criteria), but do not build the RPM package and warn the user why it has not happened.

That being said I could also live with returning an 'exit 1' an forcing failure, it would ensure we do not publish anything that is not matching containment-rpm-docker criteria. I we agree on that this might be better, then I would apply the same criteria in both situations, when $IMAGE is empty and when $CONTAINER_TAG is empty.

Reply:
I would rather prefer to have an "exit 1" because I think is weird to exit as everything was ok while you have not created the rpm. However, I think is better in this commit to be consistent with what is already in the script, so I would leave things as they are and we can create a new PR returning "exit 1" if we eventually feel we can't live with "exit 0".

Thanks for the explanation, I was just wondering if there was any other reason.

In short: LGTM